### PR TITLE
13 - API Endpoint: Add member

### PR DESCRIPTION
**Pull request by Baavi**

### CODE REVIEW

File: routes/addMember.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import { db } from "../config/db.js";
+
+const router = express.Router();
+router.post("/addmember", async (_, res) => {
```
Comment (ShiroKatake) [inline routes/addMember.js]: you need to take in `(req, res)` parameter not `(_,res)`

File: routes/test.js [hunk 2]
```diff
@@ -11,8 +11,6 @@ router.get("/test", async (_, res) => {
       
       return res.status(200).json({
         status: 200,
-        data: result
```
Comment (ShiroKatake) [inline routes/test.js:14]: did we need to modify this file?

**Review by ShiroKatake — CHANGES_REQUESTED** [review]
This is good! I'm just a bit confused about the changes in the test file, but you just need to fix up the parameters for the route.

**Comment by Baavi** [comment]
Resolve issues

**Review by ShiroKatake — APPROVED** [review]
This looks good to merge! We don't need `db.end()` ln16 in addMember.js, but we can fix that in a new ticket. For now it won't work, but the logic looks fine so we should push this along

**Pull request merged**